velox/expression/tests/CMakeLists.txt

```diff
@@ -115,11 +115,20 @@ add_library(velox_expression_runner ExpressionRunner.cpp)
 target_link_libraries(velox_expression_runner velox_expression_verifier
                       velox_functions_prestosql velox_parse_parser gtest)
 
+add_library(spark_expression_runner ExpressionRunner.cpp)
+target_link_libraries(spark_expression_runner velox_expression_verifier
+                      velox_functions_spark velox_parse_parser gtest)
+
 add_executable(velox_expression_runner_test ExpressionRunnerTest.cpp)
 target_link_libraries(
   velox_expression_runner_test velox_expression_runner velox_exec_test_lib
   velox_function_registry gtest gtest_main)
 
+add_executable(spark_expression_runner_test SparkExpressionRunnerTest.cpp)
+target_link_libraries(
+  spark_expression_runner_test spark_expression_runner velox_exec_test_lib
```

Comment:
looks like spark_expression_runner library is only used for spark_expression_runner_test, why not put them all as a part of spark_expression_runner_test then.